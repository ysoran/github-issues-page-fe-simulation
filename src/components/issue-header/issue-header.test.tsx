import React from 'react';
import { shallow } from 'enzyme';
import IssueHeader from './issue-header';

const app = () => {
    const wrapper = shallow(<IssueHeader setFilter={()=>{}} />);
    return { wrapper };
};

describe('Issue Header', () => {
    it('should render snapshot', () => {
        const { wrapper } = app();
        expect(wrapper).toMatchSnapshot();
    });
});
