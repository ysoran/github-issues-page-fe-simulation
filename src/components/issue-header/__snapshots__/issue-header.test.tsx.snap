// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issue Header should render snapshot 1`] = `ShallowWrapper {}`;
